generic_config_updater/generic_updater.py

```diff
@@ -0,0 +1,469 @@
+import json
+import jsonpatch
+import sonic_yang
+import os
+import copy
+from enum import Enum
+from swsssdk import ConfigDBConnector
+from imp import load_source
+load_source('sonic_cfggen', '/usr/local/bin/sonic-cfggen')
```

Comment:
>sonic_cfggen [](http://example.com/codeflow?start=13&length=12)

this is a new dependency. please add it in install_requires in setup.py

Reply:
let me try this idea

Reply:
Was not able to include in install_requires: tried `sonic_cfggen` and `sonic-cfggen`.

Also when checking the code behind `show runningconfiguration all` it is implemeted as:
```python
@runningconfiguration.command()
@click.option('--verbose', is_flag=True, help="Enable verbose output")
def all(verbose):
    """Show full running configuration"""
    cmd = "sonic-cfggen -d --print-data"
    run_command(cmd, display_cmd=verbose)
```

Which means they just depend on the system to have, and we here are doing the same.

We can either leave it as as, or just rely on the command from cli

Reply:
The name is `sonic-config-engine`. Currently it is `tests_require`. We should remove it, and add as `install_requires`.

Reply:
oh, let me try that

Reply:
updated locally

Reply:
updated